### PR TITLE
Upstream merge

### PR DESCRIPTION
**Pull request by rwakida**

### CODE REVIEW

File: client/views/app/message.coffee [hunk 1]
```diff
@@ -34,11 +35,18 @@ Template.message.helpers
 				return this.html
 
 	system: ->
-		return 'system' if this.t in ['s', 'p', 'f', 'r', 'au', 'ru', 'ul', 'nu', 'wm', 'uj']
-
+		return 'system' if this.t in ['s', 'p', 'f', 'r', 'au', 'ru', 'ul', 'nu', 'wm', 'uj', 'rm']
```
Comment (bbrockman) [inline client/views/app/message.coffee:38]: one/two letter variables kill me!

File: client/views/app/sideNav/userStatus.coffee [hunk 2]
```diff
@@ -29,6 +29,9 @@ Template.userStatus.helpers
 			username: username
 		}
 
+	isAdmin: ->
```
Comment (bbrockman) [inline client/views/app/sideNav/userStatus.coffee:24]: Shouldn't this function be in one place?  It is used all over.

Comment (rwakida) [inline client/views/app/sideNav/userStatus.coffee:24]: Their code, not ours :wink:

Comment (bbrockman) [inline client/views/app/sideNav/userStatus.coffee:24]: I know I know.  Overall, good code.  Remember, I'm anal.

**Comment by bbrockman** [comment]
I'm concerned about this commit 'Always set oldest user as admin when no admins are found.', but I can't find where in the code they do this.

**Comment by rwakida** [comment]
https://github.com/RocketChat/Rocket.Chat/search?utf8=%E2%9C%93&q=set+oldest

**Comment by rwakida** [comment]
good catch.  That would be....bad

**Comment by bbrockman** [comment]
Why does this not come across in the diff files?  initialData.coffee isn't in the upstream-merge diff.

**Comment by rwakida** [comment]
I removed the set oldest user as admin code.

**Comment by bbrockman** [comment]
That still bothers me that it didn't show up in the diff.  Do you know why?

**Comment by rwakida** [comment]
not sure.  I can't see the initialData, but can see the v12 changes.  

how are you viewing the diff?  The github ui says there are too many diffs to show and initialData doesn't appear.

**Comment by bbrockman** [comment]
Am I crazy.  I don't see the v12.coffee file in the "Files Changed" tab.  Also searched for oldest (after reading through the files manually).

**Comment by bbrockman** [comment]
Oh damn.  I missed "Sorry, we could not display the entire diff because too many files (562) changed." at the top of the page.  Even missed it when I read your message.  I have selective reading!

**Comment by rwakida** [comment]
:smile: me too

**Comment by rwakida** [comment]
Maybe it's something I did when merging upstream.  Git still confuses me at times.

**Comment by douglasrapp** [comment]
I've noticed 2 things:

1) The left side nav no longer "slides" in. Rather, it just suddenly appears. This isn't our issue, though - I took a look at the actual Rocket Chat demo page, and the same thing happens there. Unfortunate that the animation is broken. I can't speak to what broke it.

2) When I first started the app and opened the page, when I tried to create a new direct message or private group, nothing happened, but the left nav area kind of 'froze'. I had to refresh the page, and then I could successfully bring up the dialog to create the new conversation. I can only seem to replicate the error if I log out and log in as a different user that I haven't used before. Also, when it happens, I get the following in the console:

> Exception from Tracker recompute function:
> Error: $in needs an array

**Comment by berkich** [comment]
I noticed that if you click on My Account then change Avatar the right
panel shows the avatar selections. Then click the up arrow next to My
Account the left panel
shows the channels, direct messages, private groups... etc. Click on any of
the plus signs (such as create a direct message) and the left panel goes
back to 'My Account'
This happens with the profile selection also.

**Comment by douglasrapp** [comment]
I think this is ready. :+1:

**Comment by rwakida** [comment]
Fixes #71

**Pull request merged**